spec/spec.md

```diff
@@ -203,6 +203,28 @@ This profile uses certain versions of specifications that have not yet reached f
   - First Implementer's Draft of OpenID for Verifiable Prensetations specification
   - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). We will continue to use the term ID Hub rather than Decentralized Web Node to avoid confusion.
 
+### JWT-VCs
+
+#### Using JWT claims instead of their counterparts in the data model specification
+
+Section 6.3.1 of [[ref: VC Data Model v1.1]] provides two options for how to encode properties defined in VC Data Model v1.1 as a JWT. 
```

Comment:
```suggestion
Section 6.3.1 of [[ref: VC Data Model v1.1]] provides two options for how to encode properties defined in VC Data Model v1.1 as a JWT. One option is to use registered JWT claims instead of respective counterparts defined in a VC Data Model v1.1. Another option is to use JWT claims in addition to VC Data Model v1.1 counterparts.

registered JWT claim names MUST be used, instead of or in addition to, their respective standard [verifiable credential](https://www.w3.org/TR/vc-data-model/#dfn-verifiable-credentials) counterparts
```

Reply:
@selfissued better?

Reply:
imo, easier to read like this:
```suggestion
Section 6.3.1 of [[ref: VC Data Model v1.1]] provides two options for how to encode properties defined in VC Data Model v1.1 as a JWT:
  1. Use registered JWT claims instead of respective counterparts defined in a VC Data Model v1.1.
  2. Use JWT claims in addition to VC Data Model v1.1 counterparts
```